bitbots_vision/src/bitbots_vision/vision_modules/lines.py

```diff
@@ -152,7 +152,6 @@ def filter_points_with_candidates(linepoints, candidates):
         """
         filtered_linepoints = []
         for linepoint in linepoints:
-            # TODO: use list comprehension instead of map
```

Comment:
This has been done in 97d974528a60dae41e4f7604de14d875c42fa10b.
Thanks for removing the comment.